releases/releaser/releaser.go

```diff
@@ -62,13 +61,11 @@ type PublishParameters struct {
 }
 
 func (r *Releaser) Publish(ctx context.Context, params PublishParameters) error {
-	err := r.uploadBinaries(ctx, params)
-	if err != nil {
+	if err := r.uploadBinaries(ctx, params); err != nil {
```

Comment:
We don't inline errors like this in order to make full use of named returns. Even if we don't use named returns on that we maintain this style to remain consistent. Please remove these stylistic changes.

Reply:
@SemanticallyNull These are still compatible with named returns. 

When a function with named returns returns, it assigns the return values to the named returns before running the defer block.

Please look at this example: https://go.dev/play/p/2hAvO0GeYug